functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/customAssertions/BuildProjectAssert.java

```diff
@@ -0,0 +1,50 @@
+package org.arquillian.smart.testing.ftest.customAssertions;
+
+import java.io.File;
+import org.assertj.core.api.AbstractAssert;
+import org.assertj.core.api.FileAssert;
+import org.assertj.core.api.JUnitSoftAssertions;
+import org.jboss.shrinkwrap.resolver.api.maven.embedded.BuiltProject;
+import org.junit.Rule;
+
+import static org.arquillian.smart.testing.ftest.customAssertions.CustomAssertions.assertThat;
+
+public class BuildProjectAssert extends AbstractAssert<BuildProjectAssert, BuiltProject> {
+
+    @Rule
+    public static final JUnitSoftAssertions softly = new JUnitSoftAssertions();
```

Comment:
Soft assertions shouldn't be enabled by default, so it shouldn't belong here. We can always use it outside from the assertion when needed.